demisto_sdk/commands/validate/tests/PB_validators_test.py

```diff
@@ -199,6 +202,78 @@ def test_is_deprecated_with_invalid_description(content_item, expected_result):
     )
 
 
+def test_does_playbook_have_unhandled_conditions():
+    """
+    Given: A playbook with condition tasks.
+    When:
+    - Some condition option are unhandled.
+    Then:
+    - Ensure the validation failes on the expected errors only.
+    """
+    playbook = create_playbook_object()
+    playbook.tasks = {
```

Comment:
you can move the task creation into the `create_playbook_object`

Reply:
it's used in many other tests, each one has a different value for `tasks` (including the "untouched" playbook)